releasetasks/templates/firefox/routes.yml.tmpl

```diff
@@ -0,0 +1,23 @@
+# Routing Templates: releases v2!
+
+# version (simple, direct route)
+{% do release_routes.append('releases.v2.{app}.version.{version}.build{build_num}'
+        .format(app=product, version=version.replace('.', '_'), build_num=buildNumber)) %}
+# pushdate
+{% do release_routes.append('releases.v2.{app}.pushdate.{year}.{mm}.{dd}.{version}.build{build_num}'
+        .format(app=product, year=now.year, mm=now.month, dd=now.day,
+                version=version.replace('.', '_'), build_num=buildNumber)) %}
+# revision
+{% do release_routes.append('releases.v2.{app}.revision.{branch}.{revision}.{version}.build{build_num}'
```

Comment:
Also for the revision routes, I think it might make more sense to have the {revision} be right after the 'revision' namespace. So revision.{revision} rather than revision.<somethingelse>.{revision}.

In the non-release routes, it looks like we do {project}.revision.{revision}.{product}, so this comes out to something like mozilla-central.revision.abcd1234.firefox. I guess in release-world it makes more sense to have the product (which is equivalent to {app} here, right?) first since you're looking for releases by product rather than branch typically?